Apps/UnitTests/UnitTests.cpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "gtest/gtest.h"
+#include <UrlLib/UrlLib.h>
+#include <arcana/threading/task.h>
+
+namespace Babylon
+{
+
+    int test(int argc, char** argv)
+    {
+        ::testing::InitGoogleTest(&argc, argv);
+        return RUN_ALL_TESTS();
+    }
```

Comment:
Who calls this function?

Reply:
We don't need it since we are linked with gtest_main (which defines an entry point), nice catch! Removed